python/pyspark/ml/tuning.py

```diff
@@ -106,7 +107,7 @@ class CrossValidator(Estimator):
     >>> lr = LogisticRegression()
     >>> grid = ParamGridBuilder().addGrid(lr.maxIter, [0, 1]).build()
     >>> evaluator = BinaryClassificationEvaluator()
-    >>> cv = CrossValidator(estimator=lr, estimatorParamMaps=grid, evaluator=evaluator)
+    >>> cv = CrossValidator(estimator=lr, estimatorParamMaps=grid, evaluator=evaluator, seed=42)
```

Comment:
We set default value of ```seed``` to [```this.getClass.getName.hashCode.toLong```](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala#L295) at Scala side. I think we should keep consistent with that, otherwise users may get different result when training with the same dataset.